### PR TITLE
added De Morgan's Law link and fixed bold syntax

### LINKED ISSUE #8: 13-Refact: Adding link for DeMorgan's laws and fixing the incorrect bold in "Refactoring Lab Directions" section.
Adding a link about DeMorgan's laws can be helpful for students to better understand the negations.

Screen Shot 2023-03-04 at 1 35 59 PM

In the bottom of the file, there is incorrectly implementation of markdown file syntax, so the context didn't be bold as the expected. It need a small fix.
Screen Shot 2023-03-04 at 1 40 01 PM

### PR DESCRIPTION
**Pull request by Yupeng-P**
fixes #8

### CODE REVIEW

File: docs/labs/13-Refact.md [hunk 1]
```diff
@@ -575,11 +575,11 @@ Credit will be given to how much more readable,
...
 **You can work on this project in pairs,
 but should not break up or attempt to break up the work.
-Rather you should:
+Rather you should:**
 
-1. work on it together,
-2. propose ideas to each other about how to refactor
-3. propose ideas on what should be refactored.**
+**1. work on it together,**<br>
+**2. propose ideas to each other about how to refactor,**<br>
```
Comment (onzfonz) [inline docs/labs/13-Refact.md:581]: Are the `<br>` tags necessary?  I don't think they are necessary.
I'm going to have you remove them from here.

**Review by onzfonz — CHANGES_REQUESTED** [review]

File: docs/labs/13-Refact.md (hunk 1 above)
Comment (Yupeng-P) [reply docs/labs/13-Refact.md]: Just removed it

File: docs/labs/13-Refact.md [hunk 2]
```diff
@@ -575,11 +575,13 @@ Credit will be given to how much more readable,
 flexible and possibly simple the code is in relation to the original.
 **You can work on this project in pairs,
 but should not break up or attempt to break up the work.
```
Comment (onzfonz) [inline docs/labs/13-Refact.md:577]:
```suggestion
but should not break up or attempt to break up the work.**
```

File: docs/labs/13-Refact.md [hunk 3]
```diff
@@ -575,11 +575,13 @@ Credit will be given to how much more readable,
...
-Rather you should:
+Rather you should:**
```
Comment (onzfonz) [inline docs/labs/13-Refact.md:578]:
```suggestion
Rather you should:
```

**Review by onzfonz — CHANGES_REQUESTED** [review]
just commit those suggestions and you should be good

**Review by onzfonz — APPROVED** [review]

**Pull request merged**